Tasks/CopyFiles/task.json

```diff
@@ -0,0 +1,87 @@
+﻿{
+  "id": "5BFB729A-A7C8-4A78-A7C3-8D717BB7C13C",
+  "name": "CopyFiles",
+  "friendlyName": "Copy Files",
+  "description": "Copy files from source folder to target folder using minimatch patterns",
+  "helpMarkDown": "[More Information](http://go.microsoft.com/fwlink/?LinkID=)",
```

Comment:
Make the fwlink point to the same as the current copyPublish combo task we have now.  In that documentation, we can cover, copy, publish, copyPublish